common/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
+    <string name="staking_reward_estimation_bottom_sheet_title">Earnings in the network</string>
+
+    <string name="staking_only_apr">APR</string>
+    <string name="staking_only_apy">APY</string>
```

Comment:
We already have

```
   <string name="staking_apy">%s APY</string>
    <string name="staking_apr">%s APR</string>
```

Reply:
I don't need a template here, just APY and APR, 'cause I use two TextView to display "1% APY"

Reply:
Yeah, I meant that we could reuse existing strings to not to have a lot of dublicates